content/articles/creating-dummy-data-in-python-using-faker-package/index.md

```diff
@@ -0,0 +1,105 @@
+```
+For addresses we use `address()` and for names we use `name()` methods.
+#### 3. Creating same dummy data
+You may wish to use the same collection of data again. Seeding the generator is a viable option. Seeding is a method of populating a table with fictitious data. In addition, dummy data is generated using faker. So you don't have to use faker to write dummy data. The following code generates dummy data using the seed() method:
+```Python
+Faker.seed(111)
+print(ourFake.text())
+```
+Output:
+```bash
+Management huge pay college cover instead. Consumer leg start research her.
+Sound finish set draw notice imagine that. Blue between least democratic down week wait. Reduce inside me.
+```
+#### 4. Creating unique dummy data
+You can utilize the generator's.unique property to ensure that the dummy data created is unique.
+```python
+# generates 10 unique texts 
+texts = [ourFake.unique.name() for b in range(10)]
+```
+#### 5. Command-line usage of Faker package
+Command-line invocation of the faker package allows you to directly type in the command prompt code.
+
+#### 6. Creating localized dummy data
+The Faker Generator may generate localized false data if a place is provided as an input. Moreover, it can be translated into a wide range of languages. Otherwise, all locales must use the python list data type. 
+
+The following are some names we can come up with:
+```Python
+ourFake = Faker()
+for b in range(10):
+    print(ourFake.name())
+```
+Output:
+```bash
+Daniel Davidson
+Kristin Stewart
+Derrick Tran
+Matthew Mccarty
+Kevin Davis
+Kim Watkins
+Ashley Humphrey
+Corey Webb
+Melissa Barrera
+Juan Greene
+```
+### Other dummy data creation methods
+They are as follows:
+
+1. In Python, Numpy's Random module can be accessed: Pseudorandom numbers can be generated with the random package. Many functions are available, including `rand()`, `randint()`, and many more.
+**WExample**
+```python
+import numpy as num
+myArray = num.random.rand(5)
+print("Array : \n", myArray);
+```
+Output:
+```bash
+Array : 
+ [0.02471149 0.41561035 0.76783821 0.89628689 0.8540258 ]
+```
+2. Fauxfactory: Automated testing may be made easier with FauxFactory's random data generator!. When building tests for your application, you may need to provide the sections you're testing with random, non-specific data. To test your code quickly, you can use this anytime. [Here](https://fauxfactory.readthedocs.io/en/latest/#), you may learn more about it.
+###  A few more words regarding dummy data
+As a result of dummy variables, a single regression equation like y= a + bc might represent multiple groups. The term dummy variable refers to a numerical variable that is used to represent categorical information. The necessity for different equations for each subgroup is eliminated. Putting it in another way, dummy variables are like switches in an equation that can enable or disable other variables.

```

Comment:
`As a result of dummy variables, a single regression equation like y= a + bc might represent multiple groups`

What do you mean?

Reply:
`The term dummy variable refers to a numerical variable that is used to represent categorical information. The necessity for different equations for each subgroup is eliminated. Putting it in another way, dummy variables are like switches in an equation that can enable or disable other variables.`

This looks very unclear and I'm finding it difficult to comprehend it. Please rephrase them.